### PR TITLE
add window-ops

### PR DESCRIPTION
**Pull request by jose-moralez**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/window_ops/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - pip
+  - python >=3.6
+  - packaging
+  - numba
+  - numpy
+test:
+  imports:
+  - window_ops
+about:
+  dev_url: https://jose-moralez.github.io
```
Comment (dopplershift) [inline recipes/window_ops/meta.yaml:28]: These URLs don't actually work (they're missing the `window_ops` part of the URL.

File: recipes/window_ops/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  sha256: dc56e64b361533e659b339a940405e8c6582aab31a822e12485db8239de82804
+  url: https://files.pythonhosted.org/packages/09/cf/90624db95c1797d6db0686318152d74d941a893d2f9a1da731e5f434e3fd/window_ops-0.0.12.tar.gz
+build:
+  noarch: python
+  number: '0'
+  script: '{{ PYTHON }} -m pip install . -vv'
+requirements:
+  host:
+  - pip
+  - python >=3.6
```
Comment (dopplershift) [inline recipes/window_ops/meta.yaml:14]: Only `pip` and `python` should be necessary in `host`

File: recipes/window_ops/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - packaging
+  - numba
+  - numpy
+  run:
+  - pip
+  - python >=3.6
+  - packaging
```
Comment (dopplershift) [inline recipes/window_ops/meta.yaml:21]: `packaging` doesn't seem to be used?

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/window_ops/meta.yaml (hunk 1 above)
Comment (jose-moralez) [reply recipes/window_ops/meta.yaml]: Thank you! I've fixed them in c85ca92

File: recipes/window_ops/meta.yaml (hunk 2 above)
Comment (jose-moralez) [reply recipes/window_ops/meta.yaml:14]: Removed the other imports in c85ca92

File: recipes/window_ops/meta.yaml (hunk 3 above)
Comment (jose-moralez) [reply recipes/window_ops/meta.yaml]: Removed in c85ca92

**Review by dopplershift — APPROVED** [review]

**Pull request merged**